errors/language.am

```diff
@@ -8,6 +8,7 @@
 
 ## This file is generated by scripts/source-maintenance.sh.
 
+if ENABLE_TRANSLATION
```

Comment:
As mentioned on the line directly above this file is auto-generated.

These changes need to be made to the generator script **OR** (if it works) wrapped around the am_include statement that pulls this file in.

Reply:
Fixed in b5dae49. Sorry I missed this. The file I edited did not have the "generated" warning, and I missed it when reviewing the upport to master.